.circleci/config.yml

```diff
@@ -63,15 +63,17 @@ jobs:
               JUNIT_REPORT_DIR=./test-results SCREENSHOT_DIR=./screenshots TEST_ENV=sandbox npm run test-checkout-magento1
             fi
       - store_test_results:
-          path: ./test-results
+          path: ./integration-tests/test-results
       - store_artifacts:
-          path: ./screenshots
+          path: ./integration-tests/screenshots
```

Comment:
Can you update the line `mkdir -p ./screenshots` above?

Reply:
there is no need for that, mkdir is already happening inside /integration-tests dir

Reply:
Gotcha, I see that now.